test/api/v3/integration/quests/POST-groups_groupId_quests_force-start.test.js

```diff
@@ -149,5 +149,86 @@ describe('POST /groups/:groupId/quests/force-start', () => {
       expect(partyMemberThatIgnores.party.quest.key).to.not.exist;
       expect(partyMemberThatIgnores.party.quest.completed).to.not.exist;
     });
+
+    it('removes users who have not accepted the quest from quest.members', async () => {
+      let partyMemberThatRejects = partyMembers[1];
+      let partyMemberThatIgnores = partyMembers[2];
+
+      await leader.post(`/groups/${questingGroup._id}/quests/invite/${PET_QUEST}`);
+      await partyMembers[0].post(`/groups/${questingGroup._id}/quests/accept`);
+      await partyMemberThatRejects.post(`/groups/${questingGroup._id}/quests/reject`);
+
+      await leader.post(`/groups/${questingGroup._id}/quests/force-start`);
+
+      await Bluebird.delay(500);
```

Comment:
If you look at the force start code it's not all synchronous, it kicks things off purposefully without waiting for them. If you look at the other tests in this file, the majority of them also have this.